### PR TITLE
feat(pricing, utils, types): adds money amount to pricing module

### PR DESCRIPTION
**Pull request by riqwan**
What:

- Adds money amount service / repo / model
- Adds money amount to entry service
- Adds tests for services
- Refreshes schema
- Update joiner config to include money amounts

RESOLVES CORE-1478
RESOLVES CORE-1479

### CODE REVIEW

File: packages/types/src/pricing/service.ts
Comment (olivermrbl) [inline packages/types/src/pricing/service.ts]:
```suggestion
     id: string,
```

**Review by olivermrbl — APPROVED** [review]
LGTM - pending @carlos-r-l-rodrigues's approval 🔥

**Review by carlos-r-l-rodrigues — APPROVED** [review]
It is missing the `softDelete` and `restore` methods, but that's alright to have them in another PR.

LGTM

**Comment by riqwan** [comment]
@carlos-r-l-rodrigues thanks, i'll create a ticket and get to it soon.

**Pull request merged**
